src/pages/guides/app_builder_guides/storage/database.md

```diff
@@ -0,0 +1,621 @@
+---
+keywords:
+  - Adobe I/O
+  - App Builder
+  - Database Storage
+  - aio-lib-db
+  - DocumentDB
+  - MongoDB
+title: Database Storage Getting Started
+---
+
+# Overview
+
+Database Storage for App Builder provides document-style database persistence for AIO Runtime Actions. The **aio-lib-db**, which is closely modeled on the MongoDB Database Driver for NodeJS, provides the primary programming interface, while the DB Plugin in the AIO CLI provides additional access.
+
+There is a strict one-to-one relationship between an AIO Project Workspace and a Workspace Database, and each Workspace Database is entirely isolated from all other Workspace Databases.
+
+Before use, a Workspace Database must be explicitly provisioned using either **aio-lib-db** or the AIO CLI. This is a self-service operation subject to organizational quotas and limits.
+
+Workspace Databases are provisioned in one and only one of the following regions: `amer`, `apac`, and `emea`. These acronyms are defined as follows:
+
+- `amer`: North, Central, and South America. Data is stored in the US.
+- `apac`: Asia and Pacific. Data is stored in Japan.
+- `emea`: Europe, the Middle East, and Africa. Data is stored in the EU.
+
+## DB plugin in the AIO CLI
+
+The DB Plugin in the AIO CLI is a utility that facilitates provisioning, initializing, querying, and monitoring Workspace Databases.
+
+The following is only a brief introduction to the DB Plugin. For more thorough documentation see [aio-cli-plugin-app-storage](https://github.com/adobe/aio-cli-plugin-app-storage/tree/epic/abdb-implementation).
+
+### Installation
+
+To be included in aio --version >= 11.0.0
```

Comment:
Could link to or reproduce the aio install command

Reply:
Not sure this should be included here, but prior to GA the following command is needed to install the db plugin:
```
aio plugins:install @adobe/aio-cli-plugin-app-storage@next
```